scripts/utils.mk

```diff
@@ -35,20 +35,15 @@ lint-fix:
 .PHONY: lint-fix
 
 ## vet: Run go vet
-vet: 
+vet:
 	@echo "--> Running go vet"
 	@go vet ./...
 .PHONY: vet
 
 ## mock-gen: generate mocks of external (commetbft) types
```

Comment:
can we fix the typo while modifying this :D

Reply:
done